deps/verifier/src/cca/remote.rs

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2023 Arm Ltd.
+// Copyright (c) 2025 Linaro Ltd.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use super::*;
+use base64::Engine;
+use config::Config;
+use core::result::Result::Ok;
+use ear::Ear;
+use log::{debug, error};
+use std::str;
+
+const MEDIA_TYPE: &str = r#"application/eat-collection; profile="http://arm.com/CCA-SSD/1.0.0""#;
+
+fn evidence_builder(
+    nonce: &[u8],
+    accept: &[String],
+    token: Vec<u8>,
+) -> Result<(Vec<u8>, String), veraison_apiclient::Error> {
+    debug!("server challenge: {:?}", nonce);
+    debug!("acceptable media types: {:#?}", accept);
+    // TODO: Get the CCA media type from the slice of `accept`.
+    Ok((token, MEDIA_TYPE.to_string()))
+}
+
+pub async fn verify(
+    config: Config,
+    token: &[u8],
+    expected_report_data: &Vec<u8>,
+) -> Result<Ear, anyhow::Error> {
+    let config::CcaVerifierConfig::Remote { address, ca_cert } = config.cca_verifier else {
+        panic!("expecting remote configuration")
+    };
+
+    let mut disco_builder = DiscoveryBuilder::new().with_base_url(address.clone());
+
+    if ca_cert.is_some() {
+        disco_builder = disco_builder.with_root_certificate(ca_cert.clone().unwrap())
+    }
```

Comment:
We can use to avoid `unwrap` here

```rust
if let Some(ca_cert) = ca_cert {
    disco_builder = disco_builder.with_root_certificate(ca_cert.clone());
}
```